map.html

```diff
@@ -135,6 +133,15 @@ <h5>Map of Calgary</h5>
             console.log(data);
           });
 
+          var i;
+          for (i = 0; i < data['features'].length; i++) {
+
+            pin_coords = data['features'][1]['geometry'][1]['coordinates'];
+            pin_values = "https://data.calgary.ca/resource/c2es-76ed.geojson?$where=coordinates = " + pin_coords + "&$select=issueddate,workclassgroup,contractorname,communityname,originaladdress";
+
+            ?? .bindPopup(pin_values).openPopup();
+          }
+            

```

Comment:
This part should be moved within ajax when you get the response!
you can bind the popup to the markers for clustering, see [here](https://github.com/Leaflet/Leaflet.markercluster#examples)